.github/workflows/build-tags.yml

```diff
@@ -1,46 +0,0 @@
----
```

Comment:
I'm pretty sure the one from master is used.

Reply:
I'm not sure that's true, as I remember backporting these files to get proper packages for existing branches. I tried to find a definitive answer in GitHub's documentation but while I _think_ I'm right I cannot find a clear answer.

Reply:
I also looked in the docs today and found a 'main branch' note on some *other* events. No clarity about this one. I'll put it the file back.

Reply:
`on: push` is based on the current commit that's pushed

`on: pull_request` is based on the merge of the base and head branches

`on: pull_request_target` is based on the base branch

`on: issue_comment`, `on: schedule` and a couple of other odd ones are based on the default branch

Reply:
So we need to keep our stable branches up-to-date as far as workflows are concerned, except for the last events, if I reading this right?

Reply:
right, so the yml that picks up the push tag event needs to be on the right branch, but then we quickly invoke the actual building workflows from master, so it's not to obad

Reply:
You definitely need to keep workflows current on any branches that you have, otherwise if there are security risks in them, you're at risk.

You can rely on:
`on: workflow_dispatch` and have that take an input, and have your other workflows do an `if: ...` to cause them not to do work on branches if you're so inclined.